### DISCUSSION TITLE
请教下最近看mt 引入x-api-key, 你是怎么知道要用这个key的.

### DISCUSSION THREAD

**Discussion by yipeng-yang** [jxxghp/MoviePilot#2040] [讨论]
我为了拿torrent下载地址, 搞了半天没成功, 最后还是参考你的代码用了这个key.
我是好奇你咋知道要用这个key的, 从web上我是没看到相关信息

**Comment by yipeng-yang** [comment]
这个问题我搞定了, 这个x-api-key不是必须的. 只要其它部分正常, 也可以get torrent actualy download address.

Thanks any way for open source.

**Reply by yipeng-yang** [reply]
欢迎大家多交流

**Comment by cddjr** [comment]
mt有开发文档呀，你为什么要自己研究api
